.claude/commands/generate-eng-docs.md

```diff
@@ -0,0 +1,279 @@
+---
+name: generate-eng-docs
+description:
+  Generate engineering documentation for a Nimbus component or pattern following the
+  approved team template
+command: |
+  Generate engineering documentation for the {{NAME}} {{TYPE}} following the EXACT structure approved by the team.
+  
+  Determine the type by checking:
+  - If path contains 'patterns/', then TYPE = 'pattern'
```

Comment:
Why not just explicitly tell it the {{name}} and {{type}} of the component it's documenting instead of having it analyze files?